packages/metro/package.json

```diff
@@ -53,7 +53,6 @@
     "babel-preset-react-native": "^4.0.0",
     "babel-register": "^6.24.1",
     "babylon": "^6.18.0",
-    "babylon7": "npm:babylon@^7.0.0-beta",
```

Comment:
You should still add a dependency from `metro` to `metro-babylon7`:

```
"metro-babylon7": "0.27.0",
```

Reply:
Opps, thanks! Fixed!